rust/ql/src/queries/security/CWE-079/XSSGood.rs

```diff
@@ -0,0 +1,23 @@
+use actix_web::{web, HttpResponse, Result};
+use askama::Template;
+
+// GOOD: Manual HTML encoding using an `html_escape` function
+async fn safe_handler_with_encoding(path: web::Path<String>) -> impl Responder {
+    let user_input = path.into_inner();
+    let escaped_input = html_escape(&user_input);
```

Comment:
It might be better to use a library provided HTML escaping function, since (1) that is easier for users to copy and get right and (2) it will very likely be picked up as a recommendation by autofix, making fix suggestions more consistent / reliable.

Alternatively if writing an `html_escape` function is really straightforward, I think we should provide the source code for it in the example.